package.json

```diff
@@ -49,7 +49,10 @@
     "new:util": "hygen util new",
     "prepush": "npm run test:ci",
     "static": "serve -s build",
+    "deps:graph": "depcruise --validate --output-type dot -- src | dot -T svg > dependency-graph.svg",
```

Comment:
When I run this script I get this error:
```
▶ npm run deps:graph

> crema-app-web@0.1.0 deps:graph /Users/Rob_LaFeve/Dev/crema-app-web
> depcruise --validate --output-type dot -- src | dot -T svg > dependency-graph.svg

sh: dot: command not found

  ERROR: write EPIPE
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! crema-app-web@0.1.0 deps:graph: `depcruise --validate --output-type dot -- src | dot -T svg > dependency-graph.svg`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the crema-app-web@0.1.0 deps:graph script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Rob_LaFeve/.npm/_logs/2020-01-10T18_37_40_936Z-debug.log
```

An `svg` is produced but it is blank.

Reply:
Looks like a Graphviz issue?

Reply:
@roblafeve Did you have Graphviz installed at the time of running this?

Reply:
I did not. I ran `brew install graphviz` and then it worked as expected. I simply missed your note on the script ;) I think this is good 👍